rever/activities/bibtex.xsh

```diff
@@ -0,0 +1,75 @@
+"""Activity for bumping version."""
+import datetime
+import os
+
+from rever import vcsutils
+from rever.activity import Activity
+try:
+    import bibtexparser
+except ImportError:
+    pass
+
+# $ACTIVITIES = ['bibtex']
+
+def render_authors(authors):
+    """Parse a list of of tuples of authors into valid bibtex
+
+    Parameters
+    ----------
+    authors: list of str
+        The authors eg ['Your name in nicely formatted bibtex'].
+        Please see ``<http://nwalsh.com/tex/texhelp/bibtx-23.html>`` for
+        information about how to format your name for bibtex
+
+    Returns
+    -------
+    str:
+        Valid bibtex authors
+    """
+    if len(authors) == 1:
+        return ' '.join(authors[0])
+    else:
+        return ' and '.join(authors)
+
+
+class BibTex(Activity):
+    """Writes a BibTex reference for the version of software
+
+    Minimal ``rever.xsh``::
+        '''
+        $PROJECT_NAME = <my_project>  # The name of your project
+        $AUTHORS = ['Name1', 'Name2']  # The name of the authors
+        $URL = <URL to Project>  # A URL to the code
```

Comment:
So I am not a fan of these names because they aren't namespaced by the activity name., eg `$BIBTEX_URL`.  Other activities will presumably also have URLs, authors, etc.   You should instead rely on these parameters being passed to _func() as kw/args.  Doing so, rever will bind $BIBTEX_URL to a `url` argument.

Reply:
Rever does this automatically?

Reply:
That's awesome.

Reply:
Yes, that is one of the benefits of using rever :).  You just need to write the entry point function like you would write any other function